plugins/Toolbox/src/Toolbox.py

```diff
@@ -757,6 +764,10 @@ def pluginsAvailableModel(self) -> PackagesModel:
     def pluginsInstalledModel(self) -> PackagesModel:
         return self._plugins_installed_model
 
+    @pyqtProperty(QObject, constant=True)
```

Comment:
Should this be constant? Some of the properties of these packages can change, such as `is_active`. I'm not sure how this works by reference.

I guess a test will tell. I'll accept this for now as long as we do test whether it's correctly displayed that a plug-in is inactivated.

Reply:
Turns out that this works fine. It's probably getting the model by reference and inside of that model things update, but QML is listening to those updates fine.